com.unity.render-pipelines.universal/ShaderLibrary/SSAO.hlsl

```diff
@@ -15,9 +15,14 @@ SAMPLER(sampler_BaseMap);
 SAMPLER(sampler_ScreenSpaceOcclusionTexture);
 
 // Params
-float4 _BlurOffset;
 float4 _SSAOParams;
 float4 _SourceSize;
+float4 _ProjectionParams2;
+float4x4 _CameraViewProjections[2]; // This is different from UNITY_MATRIX_VP (platform-agnostic projection matrix is used). Handle both non-XR and XR modes.
```

Comment:
Yes, but how is it different  from unity_CameraProjection * unity_WorldToCamera?

Reply:
You could do this multiplication in shader code, but I would not recommend it.
I spent years optimizing post processing effects, and moving that computation out of the shader would be the first thing we do on day 1 of optimization task. It shouldn't be so much work to provide `unity_CameraViewProjection` instead.